server/routerlicious/kubernetes/routerlicious/templates/fluid-configmap.yaml

```diff
@@ -53,10 +53,24 @@ data:
             "maxMessageSize": "16KB",
             "maxNumberOfClientsPerDocument": {{ .Values.alfred.maxNumberOfClientsPerDocument }},
             "throttling": {
-                "maxRequestsPerMs": {{ .Values.alfred.throttling.maxRequestsPerMs }},
-                "maxRequestBurst": {{ .Values.alfred.throttling.maxRequestBurst }},
-                "minCooldownIntervalInMs": {{ .Values.alfred.throttling.minCooldownIntervalInMs }},
-                "minThrottleIntervalInMs": {{ .Values.alfred.throttling.minThrottleIntervalInMs }}
+                "requests": {
```

Comment:
HttpRequests to be more explicit? or may be just https

Reply:
Good call out. I think I'll go with "restEndpoints" 😃